etc/automount.conf

```diff
@@ -0,0 +1,6 @@
+# configure mount directory
+MNT_PREFIX='/Volumes'
+# use nice names by default
+NICENAMES='YES'
+# disable automatic opening of file manager
+FM=''
```

Comment:
This may eventually be configured to automatically open up [Filer](https://github.com/ravynsoft/filer) if desired.